drivers/sensor/bq2415x/bq2415x.h

```diff
@@ -0,0 +1,153 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/*
+ * bq2415x charger driver
+ *
+ * Copyright (C) 2011-2013  Pali Rohár <pali@kernel.org>
+ * Copyright (C) 2021  Filip Primozic <filip.primozic@greyp.com>
+ */
+
+#ifndef BQ2415X_CHARGER_H
+#define BQ2415X_CHARGER_H
+
+#include <stdint.h>
+#include <kernel.h>
+
+#define BQ241X_VENDER_CODE 0x0A /* Default vender code (in binary) */
```

Comment:
s/vender/vendor/
Please fix throughout

Reply:
![image](https://user-images.githubusercontent.com/82023738/115228687-5dcaff00-a112-11eb-841e-b593a6634a97.png)
This is the way it is written in datasheet. Do you still want me to change `e` to `o`?